### ISSUE TITLE
[richieste] approvate automaticamente e UI

### ISSUE THREAD

**Issue by AlfioEmanueleFresta** [CroceRossaItaliana/jorvik#574]
Le richieste approvate automaticamente in UI non mostrano in alcun modo il fatto che la richiesta e' stata approvata automaticamente. Poi ci sta li' un `None`, con una spunta, che e' meraviglioso.

![image](https://cloud.githubusercontent.com/assets/621062/23582548/a53c5df8-0124-11e7-8e93-dc5f2be34288.png)

**Comment by luca-dex** [comment]
Concordo. Purtroppo tutta questa parte non è stata testata visto che non si riusciva a simulare il comportamento del sistema di approvazioni automatiche su luke.
